### PR TITLE
Add SERVER_TAG feature

### PR DESCRIPTION
**Pull request by DL6ER**
Fixes https://github.com/yubiuser/docker-event-monitor/issues/35

``` yml
  environment:
    SERVER_TAG: 'ryzen'
```

Screenshot from 2023-09-23 14-23-22

### CODE REVIEW

**Review by yubiuser — CHANGES_REQUESTED** [review]
Could you move the whole section

```
	// If there is a server tag, add it to the title
	if len(glb_arguments.ServerTag) > 0 {
		title = "[" + glb_arguments.ServerTag + "] " + title
	}
```

into `processEvent` and call `sendNotifications(message, title)`

I guess we'll change the title probably soon a bit more and `sendNotifications()` should not deal with modifications. All this should happen before.

**Comment by yubiuser** [comment]
Note to myself: finish the housekeeping PR which should also deal with PRs from forks...

**Comment by DL6ER** [comment]
@yubiuser I added it into `sendNotifications()` to ensure it is always added. `processEvent()` only handles "New Docker Event" messages but there is also "Starting docker event monitor" (`main()`) and there may be more coming in the future where it could easily be forgotten. The place where I added it seemed the most obvious to do it without any code duplication.

File: Readme.md [hunk 1]
```diff
@@ -97,4 +97,5 @@ Configurations can use the CLI flags or environment variables. The table below o
 | `--mailport`          | `MAIL_PORT`             | `587`   | |
 | `--mailhost`          | `MAIL_HOST`             | `""`    | `smtp@provider.com` |
 | `--filter`            | `FILTER`                | `""`    | Filter events. Uses the same filters as `docker events` (see [here](https://docs.docker.com/engine/reference/commandline/events/#filter))    |
-| `--loglevel`          | `LOG_LEVEL`             | `"info"`| Use `debug` for more verbose logging` |
+| `--loglevel`          | `LOG_LEVEL`             | `"info"`| Use `debug` for more verbose logging |
+| `--servertag`         | `SERVER_TAG`            | `""`    | Useful when running docker-event-monitors on multiple machines |
```
Comment (yubiuser) [inline Readme.md:101]:
```suggestion
| `--servertag`         | `SERVER_TAG`            | `""`    | Prefix to include in the title of notifications. Useful when running docker-event-monitors on multiple machines |
```

**Review by yubiuser — APPROVED** [review]

**Pull request merged**